presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -241,6 +241,7 @@ void PrestoServer::run() {
       address_ = fmt::format("[{}]", address_);
     }
     nodeLocation_ = nodeConfig->nodeLocation();
+    nodePoolType_ = systemConfig->poolType();
```

Comment:
What are the possible values for pool Type ? Might be good to validate the value.

Reply:
The accepted values are [here](https://github.com/prestodb/presto/blob/a766a3a1ecd5fa45426977bc05e707da5d1c138c/presto-spi/src/main/java/com/facebook/presto/spi/NodePoolType.java#L16) Sure I can add that